### PR TITLE
Add ability to install a released traffic-manager to the integration_tests.

### PR DESCRIPTION
**Pull request by thallgren**
Adds the method `InstallTrafficManagerVersion` to the integration_test framework so that tests can be written that exercises the current client against older traffic-managers.

### CODE REVIEW

File: integration_test/itest/cluster.go [hunk 1]
```diff
@@ -489,6 +537,20 @@ func (s *cluster) InstallTrafficManager(ctx context.Context, values map[string]s
 	return err
 }
 
+func (s *cluster) pullHelmChart(ctx context.Context, version string) (string, error) {
+	if err := Run(ctx, "helm", "repo", "add", "datawire", "https://app.getambassador.io"); err != nil {
+		return "", err
+	}
```
Comment (josecv) [inline integration_test/itest/cluster.go:543]: if the repo already exists this call will fail; maybe check the error?

**Review by josecv — CHANGES_REQUESTED** [review]
lgtm just a note on the helm repo add

File: integration_test/itest/cluster.go (hunk 1 above)
Comment (thallgren) [reply integration_test/itest/cluster.go:543]: I don't think that's true. You do get a message `datawire already exists with the same configuration, skipping`, but it doesn't fail.

Comment (josecv) [reply integration_test/itest/cluster.go:543]: hmm okay, i've seen it fail in the makefile, but maybe that's something else

**Review by josecv — APPROVED** [review]

**Pull request merged**